File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/1031/BusinessApplication/Models/User.partial.cs
namespace $safeprojectname$.Web
{
    using System;
    using System.ComponentModel;

    /// <summary>
    /// Erweiterungen der Klasse <see cref="User"/>.
    /// </summary>
    public partial class User
    {
        #region Macht DisplayName bindbar

        /// <summary>
        /// Überschreiben der Methode <c>OnPropertyChanged</c>, die Benachrichtigungen für die Änderung der Eigenschaft erzeugt, wenn sich <see cref="User.DisplayName"/> ändert.
        /// </summary>
        /// <param name="e">Die Ereignisargumente für die Änderung der Eigenschaft.</param>
        protected override void OnPropertyChanged(PropertyChangedEventArgs e)
        {
            base.OnPropertyChanged(e);

            if (e.PropertyName == "Name" || e.PropertyName == "FriendlyName")
            {
                this.RaisePropertyChanged("DisplayName");
            }
        }
        #endregion
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/1031/BusinessApplication/Models/User.partial.vb
Imports System
Imports System.ComponentModel

Namespace Web

    Partial Public Class User
#Region "Macht DisplayName bindbar"

        ''' <summary>
        ''' Überschreiben der Methode <c>OnPropertyChanged</c>, die Benachrichtigungen für die Änderung der Eigenschaft erzeugt, wenn sich <see cref="User.DisplayName"/> ändert.
        ''' </summary>
        ''' <param name="e">Die Ereignisargumente für die Änderung der Eigenschaft.</param>
        Protected Overrides Sub OnPropertyChanged(ByVal e As System.ComponentModel.PropertyChangedEventArgs)
            MyBase.OnPropertyChanged(e)

            If e.PropertyName = "Name" Or e.PropertyName = "FriendlyName" Then
                Me.RaisePropertyChanged("DisplayName")
            End If
        End Sub
#End Region
    End Class
End Namespace